docs/src/docs/asciidoc/contributing.adoc

```diff
@@ -1,7 +1,7 @@
 [[contributing]]
 == Contributing
 
-Spring REST Docs is intended to make is easy for you to produce high-quality documentation
+Spring REST Docs is intended to make easy for you to produce high-quality documentation
```

Comment:
Thanks. I caught that one as part of the merge. See: https://github.com/spring-projects/spring-restdocs/commit/6f4fe4ea644d77d66985c249102c8abd3d44c59d#diff-1c885f4a0e9ffe753bc7f61ba4d69f3bR4